src/main/java/com/microsoft/spring/data/gremlin/common/Constants.java

```diff
@@ -57,6 +57,12 @@
     public static final String GREMLIN_PRIMITIVE_OR = "or()";
     public static final String GREMLIN_PRIMITIVE_WHERE = "where(%s)";
 
+    public static final String GREMLIN_PRIMITIVE_VALUES = "values('%s')";
+    public static final String GREMLIN_PRIMITIVE_IS = "is(%s)";
+    public static final String GREMLIN_PRIMITIVE_GT = "gt(%d)";
```

Comment:
Why the `VALUES` is quoted with `''`, but `IS` and `GT` are not, any difference?

Reply:
`VALUES` take care of property name, it must be String with `''`. `IS` is for generate query like:
```gremlin
is(gt(2))
is(lt(3))
is(inside(0,3))
```
So it cannot be apply with `''`. If there are `IS` for String, there will be another query named `IS_STRING` with value in `is('%s')`.